src/parser.ts

```diff
@@ -234,19 +240,19 @@ export class Parser {
                 const jobMatch = /\w:/.exec(line);
                 if (jobMatch && (interactiveMatch || descriptionMatch || injectSSHAgent || noArtifactsToSourceMatch)) {
                     if (interactiveMatch) {
-                        fileSplitClone.splice(index + 1, 0, "  interactive: true");
+                        fileSplitClone.splice(index + 1, 0, "  gclInteractive: true");
                         index++;
                     }
                     if (injectSSHAgent) {
-                        fileSplitClone.splice(index + 1, 0, "  injectSSHAgent: true");
+                        fileSplitClone.splice(index + 1, 0, "  gclInjectSSHAgent: true");
                         index++;
                     }
                     if (noArtifactsToSourceMatch) {
-                        fileSplitClone.splice(index + 1, 0, "  artifactsToSource: false");
+                        fileSplitClone.splice(index + 1, 0, "  gclArtifactsToSource: false");
                         index++;
                     }
                     if (descriptionMatch) {
-                        fileSplitClone.splice(index + 1, 0, `  description: ${descriptionMatch?.groups?.description ?? ""}`);
+                        fileSplitClone.splice(index + 1, 0, `  gclDescription: ${descriptionMatch?.groups?.description ?? ""}`);
```

Comment:
We cannot do this without having to release a new major.

Reply:
which part? Did i miss something?  I was hoping to only change the variable name we use internally, 

![image](https://github.com/firecow/gitlab-ci-local/assets/28011783/66d8bca4-85f5-4060-95f2-31f9c26ff165)

Reply:
Oh, my bad. I saw the code as changing how you would construct the comments in `.gitlab-ci.yml`. 
Don't mind me :smile:

Reply:
No worries! Appreciated the careful review to ensure the stability for gcl.

Reply:
Yeah, this has a pretty big impact, so I'm being careful as well.